scripts/readmeFiles.js

```diff
@@ -0,0 +1,188 @@
+
+const skipPackages = ["helpers", "Stylers", "Tokens", "Guard", "Icon", "Overlay", "Select"];
+
+const githubIssueBody = encodeURIComponent(`
+# Help wanted
+
+## Please write your question.
+*A clear and concise description of what the question is*
+
+## Additional context
+*Add any other context or screenshots about your question here.*
+`);
+
+const renderReadmeTemplate = ({
+  displayName = "",
+  name = "",
+  description = "** require description **",
+  content = "** require content **",
+  props = "",
+  version = "",
+}) => {
+  return `
+<!-- autogenerated don't modify -->
+
+# ${name} - ${version}
+
+## Description
+${description}
+
+## Installation
+
+\`\`\`
+yarn add ${name}
+\`\`\`
+
+or with npm:
+
+\`\`\`
+npm install ${name}
+\`\`\`
+
+## Props 
+${props}
+
+<!-- autogenerated don't modify -->
+${content}
+
+## Links
+- [Storybook showcase](https://paprika.highbond.com/?path=/story/${name.split("/")[1].toLowerCase()}--showcase)
+- [Github source code](https://github.com/acl-services/paprika/tree/master/packages/${displayName}/src)
+- [Github create issue](https://github.com/acl-services/paprika/issues/new?label=[]&title=${name}%20[help]:%20your%20short%20description&body=${githubIssueBody})
+- [ChangeLog](https://github.com/acl-services/paprika/tree/master/packages/${displayName}/CHANGELOG.md)
+`;
+};
+
+const initAutoReadme = ({ path, content }) => {
+  fs.writeFileSync(
+    `${path}/README.md`,
+    `<!-- content -->
+${content}
+<!-- eoContent -->
+      `,
+    "utf8"
+  );
+};
+
+const createPropsTable = ({ info }) => {
+  if (!info.props) return "";
+
+  const table = [
+    `### ${info.displayName} \n`,
+    "| Prop  | Type  | required  | default   | Description |\n",
+    "|-------|-------| --------  | --------- | ----------- |\n",
+  ];
+
+  Object.keys(info.props).map(key => {
+    const v = info.props[key] || {};
+    let type = "-";
+    if ("type" in v) {
+      type = v.type.name === "enum" ? v.type.value : v.type.name;
+    }
+
+    const required = "required" in v ? v.required.toString() : " ";
+    const defaultValue = "defaultValue" in v ? v.defaultValue.value : "-";
+    const description = "description" in v ? v.description : " ";
+
+    return table.push(`|${key}|${type}|${required}|${defaultValue}| ${description}|\n`);
+  });
+
+  return table.join("");
+};
+
+const extractCorrectComponentDefinition = ({ desireDefinition, arrayOfComponentsDefinitions }) => {
+  const definition = arrayOfComponentsDefinitions.filter(def => def.displayName === desireDefinition);
+  return definition[0];
+};
+
+const processPropTables = ({ info, folder, path, paprikaDocs = null }) => {
+  console.log("Generating README files...", folder);
+  const propsTable = [];
+  // const autoGeneratedContent = readmeTemplate({ name, description, content: contentExtracted });
+
+  propsTable.push(createPropsTable({ info }));
+
+  // you can define on packages.json a property named paprikaDocs with an attribute subComponent to list
```

Comment:
`package.json`

Reply:
ohh you can submit a suggest commit and I can merge it :)